Add/universal login (#9038)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/sso-overview.md

pagination_prev: null
---

import UniversalLogin from '/snippets/_universal-login.md';

# Single sign-on (SSO) overview <Lifecycle status="managed, managed_plus" />

This overview explains how users are provisioned in <Constant name="dbt" /> via single sign-on (SSO).
<Constant name="dbt" /> supports JIT (Just-in-Time) provisioning and IdP-initiated login. 

<UniversalLogin />

> For SSO, users also use your account **Access URL** or **dbt Enterprise Login URL** as described in this page.

Reviewer: ```suggestion
For SSO, users also use your account **Access URL** or **dbt Enterprise Login URL**, as described in this page.
```

#### Diagram Explanation

> - **Login Page**: The user accesses the <Constant name="dbt" /> login experience (for example [https://login.dbt.com](https://login.dbt.com), your account **Access URL**, or your **dbt Enterprise Login URL**), initiating the SSO flow.

Reviewer: ```suggestion
- **Login Page**: The user accesses the <Constant name="dbt" /> login experience for example [https://login.dbt.com](https://login.dbt.com), their account **Access URL**, or their **dbt Enterprise Login URL**, and initiates the SSO flow.
```

### How should non-admin users log in?

> Non-admin users that currently login with a password can no longer do so. They must sign in using [https://login.dbt.com](https://login.dbt.com), the dbt Enterprise Login URL for your account, or an identity provider (IdP), for example, Okta or Microsoft Entra ID (formerly Azure AD).

Author: ```suggestion
Non-admin users that currently login with a password can no longer do so. They must sign in using [https://login.dbt.com](https://login.dbt.com), the dbt Enterprise Login URL for your account, or an identity provider (IdP), for example, Okta or Microsoft Entra ID.
```

> Non-admin users that currently login with a password will no longer be able to do so. They must sign in using [https://login.dbt.com](https://login.dbt.com), the dbt Enterprise Login URL for your account, or an identity provider (IdP). For example, Okta, Microsoft Entra ID (formerly Azure AD), etc.

Reviewer: ```suggestion
Non-admin users that currently login with a password can no longer do so. They must sign in using [https://login.dbt.com](https://login.dbt.com), the dbt Enterprise Login URL for your account, or an identity provider (IdP), for example, Okta or Microsoft Entra ID (formerly Azure AD).
```
